jwql/website/apps/jwql/admin.py

```diff
@@ -47,9 +47,9 @@ class RootFileInfoAdmin(admin.ModelAdmin):
 
 @admin.register(Anomalies)
 class AnomaliesAdmin(admin.ModelAdmin):
-    list_display = ('root_file_info', 'cosmic_ray_shower', 'diffraction_spike', 'excessive_saturation', 'guidestar_failure', 'persistence', 'crosstalk', 'data_transfer_error',
+    list_display = ('root_file_info', 'flag_date', 'user', 'cosmic_ray_shower', 'diffraction_spike', 'excessive_saturation', 'guidestar_failure', 'persistence', 'crosstalk', 'data_transfer_error',
```

Comment:
Testing out the new requirement that conversations are resolved before merging is allowed: 

@mfixstsci SERVER UPDATE AFTER MERGE:
run
$python manage.py makemigrations
$python manage.py migrate
$python anomaly_db_transfer.py

Reply:
This is great, thank you! Are we running these on the server, locally?